recipes/jsonextended/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "jsonextended" %}
+{% set version = "0.7.9" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 251364a566a8753c699caec696bcd0bfdfd782dc327d3292b254330670f56bfb
+
+build:
+  number: 0
```

Comment:
you can likely make this recipe a `noarch: python` recipe (see [here](https://conda-forge.org/docs/meta.html#building-noarch-packages))

Reply:
Fixed